test/helper/journey/backup_and_restore_journey.go

```diff
@@ -51,12 +51,67 @@ func backupAndRestoreJourneyTest(t *testing.T, weaviateEndpoint, backend string)
 		verifyThatAllBooksExist(t)
 	})
 
+	t.Run("verify invalid compression config", func(t *testing.T) {
+		// unknown compression level
+		b := "some-weird-config"
+		params := backups.NewBackupsCreateParams().
+			WithBackend(backend).
+			WithBody(&models.BackupCreateRequest{
+				ID:      backupID,
+				Include: []string{booksClass.Class},
+				Config: &models.BackupConfig{
+					CPUPercentage:    80,
+					ChunkSize:        512,
+					CompressionLevel: &b,
+				},
+			})
+		resp, err := helper.Client(t).Backups.BackupsCreate(params, nil)
+		helper.AssertRequestFail(t, resp, err, nil)
```

Comment:
It would be helpful to assert that this is a validation error so that we can be certain about what the user receives as an erro

Reply:
i've added to check against the error type, can't confirm the errors msgs with current testing suite, 
however i confirmed with postman 
<img width="843" alt="Screenshot 2024-01-09 at 13 50 36" src="https://github.com/weaviate/weaviate/assets/2805614/4714f7e6-072f-4a6e-afb4-42f3d50d8978">